tutorials/circuits/1_getting_started_with_qiskit.ipynb

```diff
@@ -6,22 +6,7 @@
    "source": [
```

Comment:
<p>Thanks for noticing this, but I think it's better to not to add options as this is a tutorial. We can remove <code>'mpl'</code> instead. If we do this the output style will be determined by the default user setting which I believe is 'mpl' by default but user can change by writing in a config file (can't remember the name and directory in Qiskit).</p><p> </p><pre class="ql-syntax" spellcheck="false">circ.draw()  
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>